ucp/_libs/ucx_api.pyx

```diff
@@ -386,6 +388,87 @@ cdef class UCXWorker(UCXObject):
         assert_ucs_status(status)
         return UCXEndpoint(self, <uintptr_t>ucp_ep)
 
+    cpdef ucs_status_t fence(self) except *:
+        cdef ucs_status_t status = ucp_worker_fence(self._handle)
+        assert_ucs_status(status)
+        return status
+
+    def flush(self, cb_func, cb_args, cb_kwargs=dict()):
+        cdef ucs_status_ptr_t req
```

Comment:
Would suggest using `None` as mutable default arguments (like `dict()`) sometimes have weird behavior in Python, which `None` avoids. Also checking if something is `None` in Cython is quite fast as it is just a pointer comparison. Additionally suggest making `cb_args` optional as we have done elsewhere.

```suggestion
    def flush(self, cb_func, cb_args=None, cb_kwargs=None):
        if cb_args is None:
            cb_args = ()
        if cb_kwargs is None:
            cb_kwargs = {}
        cdef ucs_status_ptr_t req
```